scripts/set-ilks-pip-whitelist

```diff
@@ -10,7 +10,13 @@ loadAddresses
 
 tokens=$(jq -r ".tokens | keys_unsorted[]" "$CONFIG_FILE")
 for token in $tokens; do
-    seth send "$(eval echo "\$PIP_${token}")" 'kiss(address)' "$MCD_SPOT"
+    osm_delay=$(jq -r ".tokens.${token} | .pip | .osmDelay" "$CONFIG_FILE")
+    addr=$(jq -r ".tokens.${token} | .pip | .addr | values" "$CONFIG_FILE")
```

Comment:
Move it up one line to use the same order as the conditional?

Reply:
I guess you mean, first define `addr`, then `osm_delay`. If that is the case, yeah, sounds good.